roles/confluent.test/molecule/confluentcli-plain-rhel/molecule.yml

```diff
@@ -0,0 +1,62 @@
+---
+driver:
+  name: docker
+platforms:
+  - name: zookeeper1
+    hostname: zookeeper1.confluent
+    groups:
+      - zookeeper
+    image: geerlingguy/docker-centos7-ansible
+    dockerfile: ../Dockerfile.j2
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent
+  - name: kafka-broker1
+    hostname: kafka-broker1.confluent
+    groups:
+      - kafka_broker
+    image: geerlingguy/docker-centos7-ansible
+    dockerfile: ../Dockerfile.j2
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent
+provisioner:
+  name: ansible
+  config_options:
+    defaults:
+      hash_behaviour: merge
+  playbooks:
+    converge: ../../../../all.yml
+  inventory:
+    group_vars:
+      all:
+        sasl_protocol: plain
+        jolokia_enabled: false
+        kafka_broker_rest_proxy_enabled: false
+        secrets_protection_enabled: true # This triggers cli installation on kafka brokers and use it to encrypt config files
```

Comment:
Thanks for investigating molecule and adding a test scenario!

But since we already have so many scenarios, I try to merge them when possible to cut down on our testing time. And since there are many scenarios that already have cli getting installed, a new one isn't really necessary. I would say just take the verify task and move it into another scenario that has secrets protection enabled or rbac enabled.

Reply:
Oh also you are getting yamllint errors, please run 
```
cd roles/confluent.test
yamllint -c ../../.yamllint ../..
```

Reply:
Actually, take a look at the `archive-community-plaintext-rhel` scenario, this would be a great one to add a test for the "tar on control node" feature you have implemented. You would need to add a `get_url` task to the prepare playbook to get the cli tar onto the control node 

```
confluent_cli_download_enabled: true

confluent_cli_archive_file_source: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/<whatever>.tar.gz"
confluent_cli_archive_file_remote: false
```

Reply:
I'll move it under `rbac-scram-custom-rhel` under `Verify - kafka_broker`. I will delete (uncommit) the additional scenario `confluentcli-plain-rhel` from `confluent.test` folder